composer.json

```diff
@@ -72,6 +100,7 @@
         "phing/phing": "2.17.3",
         "ppito/laminas-whoops": "2.2.0",
         "scssphp/scssphp": "1.6.0",
+        "seboettg/citeproc-php": "2.1.6",
```

Comment:
This is quite an old version now, but doing a straight upgrade breaks everything -- we'll have to investigate why.